polaris.shopify.com/scripts/gen-cache-json.mjs

```diff
@@ -1,14 +1,16 @@
 import path from 'path';
 import globby from 'globby';
-import {existsSync, rmSync, mkdirSync, writeFileSync, readFileSync} from 'fs';
+import {existsSync} from 'fs';
+import {mkdir, writeFile, readFile} from 'fs/promises';
```

Comment:
async-ified these commands so we can see a nice spinner in case it's going slow instead of looking like its crashed.